### PR TITLE
SISRP-12462 - Provide error messages when finaid data is unavailable

### PR DESCRIPTION
**Pull request by christianvuerings**
https://jira.berkeley.edu/browse/SISRP-12462

### CODE REVIEW

File: src/assets/templates/widgets/finaid_awards_term_widget.html [hunk 1]
```diff
@@ -3,9 +3,7 @@
     <h2>Awards by Term</h2>
   </div>
   <div data-cc-spinner-directive="finaidAwardsTerm.isLoading">
-    <div data-ng-if="errored" class="cc-widget-text">
-      An error occurred while trying to retrieve your data.
-    </div>
+    <div data-ng-if="errored" class="cc-widget-text" data-ng-include="'widgets/finaid_errored.html'"></div>
     <div data-ng-if="!errored">
```
Comment (johncrossman) [inline src/assets/templates/widgets/finaid_awards_term_widget.html]: Not `finaidAwardsTerm.errored`?

File: src/assets/templates/widgets/finaid_summary.html [hunk 2]
```diff
@@ -1,26 +1,32 @@
...
+      <div data-ng-if="finaidSummary.finaidYears.length">
+        <div data-ng-include="'widgets/finaid_summary_aidyear.html'"></div>
+        <hr data-ng-if="selected.finaidYear.availableSemesters.length" />
+        <div data-ng-include="'widgets/finaid_summary_approvals.html'"></div>
+        <div data-ng-if="canSeeFinaidData">
+          <div data-ng-include="'widgets/finaid_summary_semester.html'"></div>
+          <div data-cc-spinner-directive="finaidSummaryInfo.isLoadingData" data-ng-if="finaidSummaryInfo.isLoadedOnMainDashboard">
+            <div data-ng-include="'widgets/finaid_summary_netcost.html'" data-onload="netCostInfo.card = 'summary'"></div>
+            <hr data-ng-if="!finaidSummaryData.netCost && finaidSummaryData.fundingOffered" />
+            <div data-ng-include="'widgets/finaid_summary_fundingoffered.html'"></div>
```
Comment (johncrossman) [inline src/assets/templates/widgets/finaid_summary.html]: Not necessarily fix in this PR: The `finaid_summary_fundingoffered.html` filename is missing an underscore. Change to `finaid_summary_funding_offered.html`?

File: src/assets/templates/widgets/finaid_awards_term_widget.html (hunk 1 above)
Comment (christianvuerings) [inline src/assets/templates/widgets/finaid_awards_term_widget.html]: @johncrossman Yes, didn't change that as part of this PR

created https://jira.berkeley.edu/browse/SISRP-12479

https://github.com/christianvuerings/calcentral/blob/82eb0db4ac33ed8e7c1803d7a4ceeba913beda24/src/assets/javascripts/angular/controllers/widgets/finaid/finaidAwardsTermController.js#L20

File: src/assets/templates/widgets/finaid_summary.html (hunk 2 above)
Comment (christianvuerings) [inline src/assets/templates/widgets/finaid_summary.html]: @johncrossman opened - https://jira.berkeley.edu/browse/SISRP-12480 probably not a high priority.

**Comment by johncrossman** [comment]
thanks @christianvuerings

**Pull request merged**